src/daemon/rpc_command_executor.cpp

```diff
@@ -2032,26 +2033,37 @@ bool t_rpc_command_executor::sync_info()
     return true;
 }
 
-bool t_rpc_command_executor::get_service_node_registration_cmd(const std::vector<std::string> &args)
+bool t_rpc_command_executor::get_service_node_registration_cmd(const std::vector<std::string> &args, const cryptonote::network_type nettype)
```

Comment:
It doesn't look like this function is needed anymore? We can call directly into get_service_node_registration_cmd_raw after prepare_registration.

on_get_service_node_registration_cmd in core_rpc can be kept so you can do this over json_rpc. The other reason to keep this I can see is if you allow something like `./lokid get_service_node_registration_cmd true 100% <addr1> <amount> ...` which doesn't look like something this PR will be addressing.

Reply:
Removed the method copying the functionality into `prepare_registration`.